ddp-workspace/projects/ddp-pancan/src/assets/i18n/es.json

```diff
@@ -23,340 +23,579 @@
+            "JoinCountMeInButton": "[ES]Join Count Me In",
             "IntroductionSection": {
-                "Title": "Have a direct impact on the future of cancer research.",
-                "Text": "Count Me In is a nonprofit research initiative that engages patients and enables them to accelerate cancer research by sharing their samples, their clinical information, and their voices."
+                "Title": "[ES]Have a direct impact on the future of cancer research.",
+                "Text": "[ES]Count Me In is a nonprofit research initiative that engages patients and enables them to accelerate cancer research by sharing their samples, their clinical information, and their voices."
             },
             "LearnMoreSection": {
-                "Title": "Every patient's experience with cancer should be counted.",
-                "Text": "Patients’ medical records, tumor samples, and personal experiences hold clues—about which cancers will respond or become resistant to which therapies—that often can’t be found in research labs or clinical trials. <span class=\"italic\">Count Me In</span> enables cancer patients anywhere to share their information with researchers everywhere, to help reveal patterns in the data.",
-                "LearnMoreAboutUsButton": "Learn more about us"
+                "Title": "[ES]Every patient's experience with cancer should be counted.",
+                "Text": "[ES]Patients’ medical records, tumor samples, and personal experiences hold clues—about which cancers will respond or become resistant to which therapies—that often can’t be found in research labs or clinical trials. <span class=\"italic\">Count Me In</span> enables cancer patients anywhere to share their information with researchers everywhere, to help reveal patterns in the data.",
+                "LearnMoreAboutUsButton": "[ES]Learn more about us"
             },
             "JoinSection": {
-                "Title": "Who can join?",
-                "Text1": "Anyone who has ever been diagnosed with cancer may participate if they live in the USA or Canada.",
-                "Text2": "We welcome people of all ages, genders, races, ethnicities, and abilities."
+                "Title": "[ES]Who can join?",
+                "Text1": "[ES]Anyone who has ever been diagnosed with cancer may participate if they live in the USA or Canada.",
+                "Text2": "[ES]We welcome people of all ages, genders, races, ethnicities, and abilities."
             },
             "ParticipateSection": {
-                "Title": "Here’s how to participate",
+                "Title": "[ES]Here’s how to participate",
                 "Step1": {
-                    "Title": "STEP 1",
-                    "Time": "10-15 minutes",
-                    "Description": "Provide consent and tell us where you’ve been treated"
+                    "Title": "PASO 1",
+                    "Time": "[ES]10-15 minutes",
```

Comment:
sugggest simple translation to Spanish
`10-15 minutos` 

here and for every step below